recipe/meta.yaml

```diff
@@ -1,46 +1,42 @@
-{% set version = "1.13.0" %}
+{% set version = "2.0.0" %}
 
 package:
   name: iris
   version: {{ version }}
 
 source:
-  fn: iris-v{{ version }}.tar.gz
   url: https://github.com/SciTools/iris/archive/v{{ version }}.tar.gz
-  sha256: 2da0aa8c5d55afa8ea4fbbfed770135ddd036764b42ac31cf2d7ed206cc5961c
+  sha256: 8605ab92c67d622e83fff6d7169fe154d9f8610edd8463b697b574884c158ba2
   patches:
     # Issue reported in https://github.com/SciTools/iris/issues/2004.
     # We work around this in build.sh.
     - no_compile_setup.patch  # [osx]
 
 build:
-  number: 1
+  number: 0
 
 requirements:
   build:
     - python
-    - setuptools
+    - pip
+    - cartopy
+    - cf_units
+    - dask >=0.15.0
     - numpy
+    - pyke
     - scipy
     - six
-    - biggus >=0.14
```

Comment:
I don't have any strong opinions regarding the extras that we can add here but I thought it was time to revisit them so I used the `core.txt` as a guide to update the dependencies for now.